pages/flourish.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div>
+  <div style="height: 100%;">
```

Comment:
Please add 
`
<div style="height:100%; min-height:600px">`

So the page is handled properly in mobile browsers. Note: you may need to play about with min height. But if I re-size my current browser so it is roughly mobile size I can't see the map.